### PR TITLE
Feature/98 display email body in new tab

### PR DESCRIPTION
**Pull request by kennyevil**
## Changes in this PR

Email support interactions will now open in a new tab.

### Before

Screenshot 2021-11-29 at 14-53-51 Buy for your school

### After

Screenshot 2021-11-29 at 14-54-26 Buy for your school

Screenshot 2021-11-29 at 14-54-40 Buy for your school

### CODE REVIEW

**Comment by kennyevil** [comment]
[Format email message in event history](https://trello.com/c/cj5olgI9/98-format-email-message-in-event-history)

File: app/views/support/cases/case_history/_interaction.html.erb [hunk 1]
```diff
@@ -33,7 +33,7 @@
           <%= I18n.t("support.interaction.label.body") %>
         </dt>
         <dd class="govuk-summary-list__value">
-          <%= interaction.body %>
+          <%= interaction.show_body.html_safe %>
```
Comment (peterdavidhamilton) [inline app/views/support/cases/case_history/_interaction.html.erb:36]: The presenter could have an `email?` predicate on which we could switch, that would be nice when the HTML is extracted here

File: app/views/support/interactions/show.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<div class="govuk-body"><%= send(:markdown_to_html, @interaction.body) %></div>
```
[the remark this answers is not in the document]
Comment (kennyevil) [reply app/views/support/interactions/show.html.erb:1]: No I don't. Brainlapse on that one. I'll change it.

File: spec/presenters/support/interaction_presenter_spec.rb [hunk 3]
```diff
@@ -29,4 +29,22 @@
       ])
     end
   end
+
```
Comment (peterdavidhamilton) [inline spec/presenters/support/interaction_presenter_spec.rb:32]: I'd favour a feature spec that checked for the link being rendered and opening in a new tab. We have a custom matcher `have_link_to_open_in_new_tab`

**Review by peterdavidhamilton — CHANGES_REQUESTED** [review]

File: app/views/support/cases/case_history/_interaction.html.erb (hunk 1 above)
Comment (kennyevil) [reply app/views/support/cases/case_history/_interaction.html.erb:36]: I've added .email? as a private method on the presenter.

**Review by peterdavidhamilton — APPROVED** [review]

**Review by ryantk — APPROVED** [review]

**Pull request merged**